index.html

```diff
@@ -125,6 +125,71 @@ <h3 class="shopping-cart__title-header">Recently added items</h3>
   </div>
   <!-- / .header__cart -->
 
+    <div data-components="MyLists" class="my-lists">
+      <a class="my-lists__direct" href="#">
+        <svg class="svg-icon my-lists__icon">
+          <use xlink:href="#icon-amway-icons-my-list"></use>
+        </svg>
+        <svg class="svg-icon my-lists__indicator">
+          <use xlink:href="#icon-amway-icons-down-arrowhead"></use>
+        </svg>
+      </a>
+      <div class="my-lists__container">
+        <section class="my-lists__section">
+          <div class="my-lists__title">My Lists</div>
+        </section>
+        <section class="my-lists__section">
+          <div class="order-list">
+            <div class="order-list__title">List 1: Favourites</div>
+            <div class="order-list__details">
+              <div class="order-list__state">
+                <div class="order-list__state-item order-list__state-item--separated">5 items</div>
+                <div class="order-list__state-item order-list__state-item--separated order-list__state-item--pv">50PV</div>
+                <div class="order-list__state-item order-list__state-item--separated">$10.25</div>
+              </div>
+              <div class="order-list__state">
+                <div class="order-list__state-item js-view-order"><a href="#">View order</a></div>
+                <div class="order-list__state-item js-add-to-cart"><a href="#">Move to cart</a></div>
+              </div>
+              <div class="order-list__entries js-entries"></div>
+            </div>
+          </div>
+        </section>
+        <section class="my-lists__section">
+          <div class="order-list">
+            <div class="order-list__title">List 2: Fred Smith</div>
+            <div class="order-list__details">
+              <div class="order-list__state">
+                <div class="order-list__state-item order-list__state-item--separated">1 item</div>
+                <div class="order-list__state-item order-list__state-item--separated order-list__state-item--pv">10PV</div>
+                <div class="order-list__state-item order-list__state-item--separated">$10.25</div>
+              </div>
+              <div class="order-list__state">
+                <div class="order-list__state-item js-view-order"><a href="#">View order</a></div>
+                <div class="order-list__state-item js-add-to-cart"><a href="#">Move to cart</a></div>
+              </div>
+              <div class="order-list__entries js-entries"></div>
```

Comment:
We should remove this container because we will not implement dynamic lists right now